system/keymaps/remote.xml

```diff
@@ -577,16 +577,28 @@
     <remote>
       <left>StepBack</left>
       <right>StepForward</right>
-      <up>ChannelUp</up>
-      <down>ChannelDown</down>
+      <up>Up</up>
+      <down>Down</down>
+      <skipplus>SkipNext</skipplus>
+      <skipminus>SkipPrevious</skipminus>
+      <pageup>ChannelUp</pageup>
+      <pagedown>ChannelDown</pagedown>
```

Comment:
Okay. Will change \<pageup>ChannelUp\</pageup> to \<pageplus>ChannelUp\</pageplus> and \<pagedown>ChannelDown</pagedown> to \<pageminus>ChannelDown\</pageminus>

Is this what you want?